src/components/BooksList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
BookList is a container component as it will be connecting to the redux store to fetch books from the state, and so it should go into the `container` folder